protos/execution.proto

```diff
@@ -10,6 +10,7 @@ option go_package = "github.com/dataform-co/dataform/protos/dataform";
 message RunConfig {
   repeated string actions = 1;
   repeated string tags = 5;
+  bool include_all_tags = 10;
```

Comment:
"include all" is ambiguous in semantic meaning - IMO something like `filterTagsAsIntersection` would be better as it's more explicit.